scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -752,11 +759,14 @@ def __init__(self, params):
     ]
 
     if (self.params.staged_vars and
-        self.params.variable_update != 'parameter_server'):
+        self.params.variable_update != 'parameter_server' and
+        self.params.variable_update != 'horovod'):
```

Comment:
we sometimes use self.params.variable_update not in ('parameter_server', 'horovod') for this.

Reply:
Also - what is it used for with horovod?  staged_vars is an experimental mode that I think won't be needed with horovod, so we could just not support the combination.